netCDF4/_netCDF4.pyx

```diff
@@ -5124,6 +5124,19 @@ open for parallel access.
         ELSE:
             pass # does nothing
 
+    def get_dims(self):
+        """
+**`get_dims(self)`**
+
+return a tuple of `netCDF4.Dimension` instances associated with this 
+`netCDF4.Variable.
+        """
+        dims = ()
+        for dimname in self.dimensions:
+            dim = _find_dim(self._grp, dimname)
+            dims += (dim,)
+        return dims
```

Comment:
I think this could be much more concisely written as

```Python
return (_find_dim(self._grp, dim) for dim in self.dimensions)
```

Reply:
good idea, thanks @jhamman.  Needs to be wrapped with tuple() though.